pom.xml

```diff
@@ -197,7 +197,7 @@
     <maven.version>3.8.5</maven.version>
     <mockito.version>4.4.0</mockito.version>
     <nessie.version>${project.version}</nessie.version>
-    <nessie-apprunner.version>0.21.4</nessie-apprunner.version>
+    <nessie-apprunner.version>0.21.5</nessie-apprunner.version>
```

Comment:
We need to release nessie-apprunner after https://github.com/projectnessie/nessie-apprunner/pull/8 is merged